plexapi/video.py

```diff
@@ -89,6 +89,14 @@ def _defaultSyncTitle(self):
         """ Returns str, default title for a new syncItem. """
         return self.title
 
+    def posters(self):
+        """ Returns list of available poster objects. :class:`~plexapi.media.Poster`:"""
+        items = []
```

Comment:
I think you can do ```return self.fetchItems('%s/posters', cls=media.Poster)```

Reply:
It doesn't appear like we can. The xml for posters does not have a `filter` or `tag` to be able to use the `fetchItems`. Edit above results in a empty list.

Reply:
The for this item should be Photo, that the tag i get when i try to do this. Can change the tag and try again?

Reply:
So it worked?

Reply:
Yup. Not 100% sure why, but it does and you seem to know how/why.